addons/mail/static/src/new/thread/attachment_list.xml

```diff
@@ -26,7 +26,7 @@
                         <i class="fa fa-spin fa-spinner" />
                     </div>
                     <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
-                        <div t-if="canDelete(attachment)"
+                        <div t-if="canDelete(attachment) and props.canBeEdited"
```

Comment:
Should put `props.canBeEdited` in `canDelete(attachment)`, to cover all uses of `canDelete`